### ISSUE TITLE
Pull-to-update doesn't work right

### ISSUE THREAD

**Issue by ildar** [FredJul/Flym#438]
Update starts when the user pulls down the list. This is wrong. Should start when user removes the finger and should NOT start if he pushed back up.

**Comment by KNejad** [comment]
Yes I'm having this issue as well. Whenever I try to dismiss a story I accidentally update the feed over 3G and end up using up MB.
